week2/homework/src/index.js

```diff
@@ -1,3 +1,118 @@
 'use strict';
+let fs = require('fs');
+let order = process.argv[2];
+let orderDefinition = process.argv[3];
+let orderDefinition2 = process.argv[4];
```

Comment:
Can you come up with a better name for these variables?
When reading it in the code I have no idea what order or order definition is. And even worse what orderdefinition2 is :)
[Naming stuff is very very hard](https://martinfowler.com/bliki/TwoHardThings.html), is the hardest stuff in development 😆 

> There are only two hard things in Computer Science: cache invalidation and naming things.

Reply:
Corrected